introspection-engine/introspection-engine-tests/tests/cockroachdb/mod.rs

```diff
@@ -79,6 +79,31 @@ async fn identity_introspects_to_sequence_with_default_settings(api: TestApi) {
     expected.assert_eq(&result);
 }
 
+#[test_connector(tags(CockroachDb222))]
+async fn identity_introspects_to_sequence_with_default_settings_v_22_2(api: TestApi) {
+    let sql = r#"
+    CREATE TABLE "myTable" (
+        id   INT4 GENERATED BY DEFAULT AS IDENTITY,
+        name STRING NOT NULL,
+
+        PRIMARY KEY (id)
+    );
+    "#;
+
+    api.raw_cmd(sql).await;
+
+    let result = api.introspect_dml().await.unwrap();
+
+    let expected = expect![[r#"
+        model myTable {
+          id   Int    @id @default(sequence(maxValue: 2147483647))
```

Comment:
meh, did they intentionally introduce this new weird `maxValue`?

Reply:
Yes.